LogicAppsCustomCodeFramework/WorkflowLogicApp/.gitignore

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
I suggest create one gitignore file at the parent folder. That way, you won't need multiple ones.